dispatch/static/manager/src/js/pages/Integrations/FBInstantArticlesIntegrationPage.js

```diff
@@ -264,6 +176,9 @@ const mapDispatchToProps = (dispatch) => {
     },
     updateIntegration: (integrationId, data) => {
       dispatch(integrationActions.updateIntegration(integrationId, data))
+    },
+    resetURI: (pathname) => {
```

Comment:
you can make this 1 line case it works without the {} around the dispatch call afaik. you can leave it in if you think it makes it look cleaner/more readable.